src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2815,6 +2812,9 @@ private int[] SerializeMethodBodies(BlobBuilder ilBuilder, PdbWriter pdbWriterOp
 
             var encoder = new MethodBodiesEncoder(ilBuilder);
 
+            var mvidStringHandle = default(UserStringHandle);
```

Comment:
Yes, we can. I think that's unnecessary though, since this is just passed thru two call sites.

---
In reply to: [63592363](https://github.com/dotnet/roslyn/pull/11357#discussion_r63592363) [](ancestors = 63592363,63577698,63575659)